### PR TITLE
added name as a maintainer since revert deleted it

### PR DESCRIPTION
**Pull request by JayNasty**
Title says it all.

### CODE REVIEW

**Comment by JayNasty** [comment]
Added google analytics visitor counter successfully across chrome and firefox. Tested fine locally for me.

File: index.html [hunk 1]
```diff
@@ -179,7 +184,7 @@ <h2>Donating to the cause</h2>
             <div class="row">
               <div class="col-md-2"></div>
               <div class="col-md-8">
-                <p style="text-align: center;">&copy; 2018&ndash;2019 the UNT Grade Distributions Team: <a href="https://garrettgu.com">Garrett Gu</a>, <a href="https://jeffw.xyz">Jeffrey Wang</a>, <a href="https://www.linkedin.com/in/josephwickline/">Joseph Wickline</a>, <a href = "http://prestja.com">Jacob Preston</a>.</p>
+                <p style="text-align: center;">&copy; 2018&ndash;2019 the UNT Grade Distributions Team: <a href="https://garrettgu.com">Garrett Gu</a>, <a href="https://jeffw.xyz">Jeffrey Wang</a>, <a href="https://www.linkedin.com/in/josephwickline/">Joseph Wickline</a>, <a href = "http://prestja.com">Jacob Preston</a> and <a href=JordanHamilton@my.unt.edu">Jordan Hamilton</a>.</p>
```
Comment (jeffw16) [inline index.html]: should be mailto:jordanhamilton@my.unt.edu

File: index.js [hunk 2]
```diff
@@ -84,7 +90,7 @@ function generateQuery() {
 }
 
 function formatResult( result ) {
-    return result.subj + " " + result.num + "." + result.sect + " " + result.desc + " (" + result.prof + ") - " + result.term;
+    return result.subj + " " + result.num + "." + result.sect + "  |  " + result.desc + "  |  (" + result.prof + ")  |   " + result.term;
```
Comment (jeffw16) [inline index.js]: why?

File: styles.css [hunk 3]
```diff
@@ -24,3 +24,6 @@
       .midsection {
         display: flex  
       }
+      .col-md-5 {
```
Comment (jeffw16) [inline styles.css]: please do not override bootstrap styles

**Review by jeffw16 — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (jeffw16) [reply index.html]: with quotes surrounding the value of the `href` attribute

**Review by jeffw16 — APPROVED** [review]
LGTM

**Pull request merged**